### PR TITLE
chore: documenting technical parts of the library

### PR DESCRIPTION
**Pull request by g-cheishvili**
## Related Issue(s)

closes

## Description

Documented few important key moments in our automation, structure and plugin code

### CODE REVIEW

**Review by mikerodonnell89 — APPROVED** [review]
very helpful, thanks @g-cheishvili !

File: CREATE_RELEASE.md [hunk 1]
```diff
@@ -25,24 +25,42 @@ Or if you want to use the conventional commits to determine next version automat
...
+include all the changes from the 0.40.1 release to the latest RC release, which is 0.40.1-rc.\*. In this case you need
+to do the following:
+
+-   `git checkout v0.40.0`
+-   `git cherry-pick <commit-hash>` or make changes you want to make and commit them
+-   `npm run hotfix-release`
+
+After this, lerna determines that the next version should be `0.40.1`, and CI will create a release from the HEAD.
+CI also finds out that the `0.40.1` is newer than the latest RC release, which is `0.40.1-rc.*`, and updates the main
+branch version to `0.40.1`. Also it must be noted that the npm will mark this release as the `latest`.
```
Comment (mikerodonnell89) [inline CREATE_RELEASE.md:58]: I think this isn't always true, not the case for the ng15 hotfixes I've been doing

**Review by mikerodonnell89 — COMMENTED** [review]
see comment

File: CREATE_RELEASE.md (hunk 1 above)
Comment (g-cheishvili) [reply CREATE_RELEASE.md:58]: Basically after lerna step, it compares whether the given result is bigger than the latest `latest` and chooses bigger version for the `latest` candidate

File: VERSIONING.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Versioning in fundamental-ngx
```
Comment (droshev) [inline VERSIONING.md:1]: This can be shortened and added to [`CONTRIBUTING.md`](https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md)

**Comment by droshev** [comment]
@mikerodonnell89 @InnaAtanasova @g-cheishvili shall we keep those `.md` files in the root or we put them in a subfolder?

**Comment by g-cheishvili** [comment]
> @mikerodonnell89 @InnaAtanasova @g-cheishvili shall we keep those `.md` files in the root or we put them in a subfolder?

I think they are relatively small for now and it's ok to keep them in root, but yeah, we could move them into `docs` or something like that

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**